cli/cmdlineparser.cpp

```diff
@@ -576,6 +589,16 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
                     mSettings->plistOutput = "./";
                 else if (!endsWith(mSettings->plistOutput,'/'))
                     mSettings->plistOutput += '/';
+
+                printf("mSettings->plistOutput %s\n", mSettings->plistOutput.c_str());
```

Comment:
This printf is just some debug command I assume that you should remove?

Reply:
Yes debug output i removed it.